src/Hystrix.Dotnet.Owin/Hystrix.Dotnet.Owin.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">

```

Comment:
This csproj needs more info to properly do the Nuget release, like `PackageId`, `PackageProjectUrl`, etc. Can you add those properties? You can look at `Hystrix.Dotnet.AspNet.csproj` as an example.